lib/systems/doubles.nix

```diff
@@ -13,7 +13,7 @@ let
     "x86_64-darwin" "i686-darwin" "aarch64-darwin" "armv7a-darwin"
 
     # FreeBSD
-    "i686-freebsd13" "x86_64-freebsd13"
+    "i686-freebsd" "x86_64-freebsd"
```

Comment:
Not sure if this was discussed already before but how is ABI compatibility now preserved? I.e. how will be kernel and userland kept in sync?

Reply:
This was discussed here: https://github.com/NixOS/nixpkgs/pull/254801#issuecomment-1984834380

Reply:
Ok. Looks like the FreeBSD community is doing the same with their own tooling:

```
poudriere is a BSD-licensed utility for creating and testing FreeBSD packages. It uses FreeBSD jails to set up isolated compilation environments. These jails can be used to build packages for versions of FreeBSD that are different from the system on which it is installed, and also to build packages for i386 if the host is an amd64 system. Once the packages are built, they are in a layout identical to the official mirrors. These packages are usable by [pkg(8)](https://man.freebsd.org/cgi/man.cgi?query=pkg&sektion=8&format=html) and other package management tools.
```